platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/internal/TeacherSubtree.java

```diff
@@ -34,8 +36,8 @@ public final class TeacherSubtree implements AutoCloseable {
      * @param root
      * 		the root of the subtree
      */
-    public TeacherSubtree(final MerkleNode root) {
-        this(root, new StandardTeacherTreeView(root));
+    public TeacherSubtree(@NonNull final PlatformContext platformContext, final MerkleNode root) {
```

Comment:
Try to avoid platform context in common

Reply:
Does not belong in common, but removed since I don't want to move the reconnect classes right now.